app/Domains/Materials/Models/Policies/MaterialPolicy.php

```diff
@@ -108,4 +108,13 @@ public function review(User $user, Material $material): bool
 
         return $user->hasPermissionTo(PermissionsEnum::MATERIAL_REVIEW);
     }
+
+    public function manageMaterial(User $user, Material $material): bool
```

Comment:
`Material` jest tu nadmiarowe. To jest material policy, więc samo przez się rozumie, że próbujemy zarządzać materiałem.
```suggestion
    public function manage(User $user, Material $material): bool
```